### PR TITLE
Use a linter

### PR DESCRIPTION
**Pull request by xgmak94**

### CODE REVIEW

File: client/src/components/roulette/RouletteWheel.jsx [hunk 1]
```diff
@@ -112,6 +103,7 @@ export default function RouletteWheel({
 
   useEffect(winningEffect, [winState]);
 
+  console.log(betInfo);
```
Comment (corneliusrenken) [inline client/src/components/roulette/RouletteWheel.jsx:106]: confirmed Gary doesn't use a linter

File: client/src/components/shared/WinningEffect.jsx [hunk 2]
```diff
@@ -1,34 +1,33 @@
...
 import styled from 'styled-components';
 
-
 export default function WinningEffect() {
   return (
     <WinningEffectContainer>
-      Congradulation!! You won!!
-    <Confetti/>
-    <WinningBackground />
+      Congratulations!! You won!!
```
Comment (corneliusrenken) [inline client/src/components/shared/WinningEffect.jsx:8]: 👎

**Review by corneliusrenken — APPROVED** [review]
A gift has been bestowed upon us

**Comment by corneliusrenken** [comment]
Thanks for doing this @xgmak94

**Review by Jesse132 — APPROVED** [review]
Looks great. Thank you.

**Pull request closed without merging**